### DISCUSSION TITLE
ketchup and mustard

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#209] [Ideas]
could be cool additions, and could result in some "deluxe" versions of burgers/dogs/other recipes that are worth more money. this involves adding a few things though.

- sugarcane crop
- mustard flower crop
- ketchup recipe: made from sugarcane and tomato
- mustard recipe: made from mustard seeds

**Comment by jeremyckahn** [comment]
Sounds like a good idea to me! 👍
